<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playVideoDialog</class>
 <widget class="QDialog" name="playVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="searchVideoFileButton">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>90</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>searchVideoFile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoFilePathLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
